src/pages/chat/components/change-avatar-card/avatar.ts

```diff
@@ -0,0 +1,40 @@
+import Block from '../../../../utils/Block';
+import avatarTmpl from './avatar.tmpl';
+import { AvatarProps } from './avatar.types';
+import store, { connect } from '../../../../utils/Store';
+import ChatController from '../../../../controllers/ChatController';
+import { Props } from '../../../../utils/helpers';
+
+class Avatar extends Block<AvatarProps> {
+    public constructor(props) {
+        super(
+            {
+                ...props,
+                events: {
+                    change: (e: Event) => this.handleChange(e)
+                }
+            },
+        );
+    }
+
+    public handleChange(e: Event) {
+        const formData = new FormData();
+
+        formData.append('avatar', (e.target as HTMLInputElement)!.files[0]);
```

Comment:
Лучше использовать `?`. `!` не предотвращает от ошибки в рантайме
```suggestion
        formData.append('avatar', (e.target as HTMLInputElement)?.files[0]);
```